### PR TITLE
Azure Cosmos DB for PostgreSQL

### PR DESCRIPTION
**Pull request by himanshu219**
## Purpose of this pull request

This pull request... 

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/integrations/microsoft-azure/azure-cosmos-db-for-postgresql.md [hunk 1]
```diff
@@ -15,7 +15,9 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 For Azure Cosmos DB for PostgreSQL, you can collect the following logs and metrics:
 
 * **PostgreSQL Server Logs**. These logs are available for every node of a cluster and can be used to identify, troubleshoot, and repair configuration errors and suboptimal performance.
-* **Audit Logs**. Audit logging of database activities is available through [pgAudit](https://www.pgaudit.org/) extension. By default, pgAudit log statements are emitted along with your regular log statements by using Postgres's standard logging facility. To learn more about the audit log format, refer to the [pgAudit documentation](https://github.com/pgaudit/pgaudit/blob/master/README.md#format).
+
+* **Activity logs**. Provides insight into any subscription-level or management group level events that have occurred in the Azure. To learn more, refer to the [Azure documentation](https://learn.microsoft.com/en-us/azure/azure-monitor/essentials/activity-log-schema).
+
```
Comment (kimsauce) [inline docs/integrations/microsoft-azure/azure-cosmos-db-for-postgresql.md:20]:
```suggestion
* **Activity logs**. Provides insight into any subscription-level or management group level events that have occurred in the Azure. To learn more, refer to the [Azure documentation](https://learn.microsoft.com/en-us/azure/azure-monitor/essentials/activity-log-schema).
```

Comment (kimsauce) [inline docs/integrations/microsoft-azure/azure-cosmos-db-for-postgresql.md:20]:
```suggestion
* **Activity logs**. Provides insight into any subscription-level or management group level events that have occurred in Azure. To learn more, refer to the [Azure documentation](https://learn.microsoft.com/en-us/azure/azure-monitor/essentials/activity-log-schema).
```

Comment (kimsauce) [reply docs/integrations/microsoft-azure/azure-cosmos-db-for-postgresql.md:20]: Removed extra line space and extraneous "the"

**Review by JV0812 — APPROVED** [review]
LGTM

**Pull request merged**